src/components/banners/Banner.tsx

```diff
@@ -7,6 +7,9 @@ import CheckCircleIcon from '@material-ui/icons/CheckCircle';
 import InfoIcon from '@material-ui/icons/Info';
 import NotificationsIcon from '@material-ui/icons/Notifications';
 import CloseIcon from '@material-ui/icons/Close';
+// Collapsible icons
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import ExpandLessIcon from '@material-ui/icons/ExpandLess';
```

Comment:
CoreUI has some native arrow up and down icon components. What do you think about using those instead of pulling from MUI?